### ISSUE TITLE
Upgrade wrench/wrench to prevent composer deprecation notices

### ISSUE THREAD

**Issue by ejunker** [jakubkulhan/chrome-devtools-protocol#25]
When running the latest version of composer there are deprecation notices for the wrench/wrench dependency

```
Deprecation Notice: Class Wrench\Tests\SslTest located in ./vendor/wrench/wrench/lib/Wrench/Tests/Util/SslTest.php does not comply with psr-0 autoloading standard. It will not autoload anymore in Composer v2.0.

Deprecation Notice: Class Wrench\Tests\Protocol\HybiPayloadTest located in ./vendor/wrench/wrench/lib/Wrench/Tests/Payload/HybiPayloadTest.php does not comply with psr-0 autoloading standard. It will not autoload anymore in Composer v2.0.
```

I see that they have a `v3.0.0-rc1` version that looks like fixes the issue. I am not sure how soon v3.0.0 will be released so it might be useful to try using the release candidate. It looks like they have been working on v3 since 2017 without a final release so I think we may be waiting a long time.

**Comment by jakobwildrain-si** [comment]
Perhaps if the wrench development is slow and the issue is not fixed before Composer v2.0 is out, might I suggest https://packagist.org/packages/warriorxk/phpwebsockets as a replacement library? It has the same dependencies, is up-to-date and actively maintained.

**Comment by jakubkulhan** [comment]
Dependency was upgraded to 2.0.10 in #24
